packages/storybook/.storybook/preview.js

```diff
@@ -11,6 +12,8 @@ export const parameters = {
   },
 }
 
+// const { theme } = createDesignSystem()
```

Comment:
Can we enable this to test it in storybook?

Reply:
Oh yes, that's is my plan to manually enable it. But the other components are not styled right now which is why I have it turned off when mergin in.